src/components/home/sections/sections.scss

```diff
@@ -8,7 +6,17 @@
   flex-direction: column;
   align-items: center;
   height: 85vh;
-
+  @media only screen and (min-width: 650px) and (min-heigh: 360px){ 
```

Comment:
You have the height and width reversed. Height is normally the larger number. Also `min-heigh` is missing a "t".